android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/DtQosPrioritySessions.java

```diff
@@ -46,10 +49,11 @@ protected static void createPrioritySession(MainActivity activity, MatchingEngin
         Log.i(TAG, "meHelper.mQosSessionId="+meHelper.mQosSessionId+" meHelper.mQosProfileName="+meHelper.mQosProfileName);
         // Create dialog with QOS profile names.
         List<String> items = new ArrayList<>();
-        items.add("LOW_LATENCY");
-        items.add("THROUGHPUT_S");
-        items.add("THROUGHPUT_M");
-        items.add("THROUGHPUT_L");
+        for (AppClient.QosSessionProfile profileName : AppClient.QosSessionProfile.values()) {
```

Comment:
Why name(), and not the enum const compare?

Reply:
Will fix.